lib/cartopy/crs.py

```diff
@@ -841,9 +841,17 @@ def __init__(self, central_longitude=0.0, central_latitude=0.0,
                 proj4_params += [('approx', None)]
         super().__init__(proj4_params, globe=globe)
 
+        # TODO: Let the globe return the semimajor axis always.
+        a = np.float(self.globe.semimajor_axis or WGS84_SEMIMAJOR_AXIS)
+        b = np.float(self.globe.semiminor_axis or a)
```

Comment:
Is there a reason for these to be `np.float()` instead of just leaving them as the values that are returned?